### ISSUE TITLE
fix italic font size

### ISSUE THREAD

**Issue by plexigras** [kencrocken/FiraCodeiScript#11]
https://github.com/open-source-ideas/open-source-ideas/issues/10#issuecomment-343961228

**Comment by alber70g** [comment]
I'm using `font-size: 1.2em` for the cursive text

**Comment by jamesattard** [comment]
Hello Alber70g, how do you specify that in VSCode? As far as I know, the editor.fontSize setting will affect both cursive and regular text.

**Comment by kencrocken** [comment]
@jamesattard There is an extension for VSCode that allows for custom css.  See, https://github.com/open-source-ideas/open-source-ideas/issues/10#issuecomment-281001309

**Comment by travisbotello** [comment]
Attributes are working, but obviously keywords class(es) have been changed and is not working anymore `.vs-dark .mtk15`

**Comment by AaronHolbrook** [comment]
That doesn't help users of IDEs that you can't adjust via css like PHPStorm.

**Comment by rodrigocfd** [comment]
Italics size is totally off... the font looks nice, but it's pretty much useless for coding.

**Comment by maciejcieslik** [comment]
Yup, there is too much difference between cursive and sans-serifs. It need to be fixed to font be usable. Hope it will be fixed as font looks nice.

**Comment by plweil** [comment]
+1

**Comment by kencrocken** [comment]
I apologize to all the people who want to use this font and it's not meeting your needs.  Unfortunately, time is fairly scarce and I haven't really been able to sit down and dive into the font.  The project has gotten pretty stale.  I am not familiar with theming for IntelliJ IDEs or really anything else besides VSCode.  

If this font is not meeting your needs, and the above-linked workaround is not suitable, I would encourage all to fork the project and tinker.
